gremlin-dotnet/src/Gremlin.Net/Structure/IO/GraphBinary/StreamExtensions.cs

```diff
@@ -59,6 +59,31 @@ public static async Task<byte> ReadByteAsync(this Stream stream, CancellationTok
             return readBuffer[0];
         }
 
+        /// <summary>
+        ///     Asynchronously writes a <see cref="sbyte"/> to a <see cref="Stream"/>.
+        /// </summary>
+        /// <param name="stream">The <see cref="Stream"/> to write the <see cref="sbyte"/> to.</param>
+        /// <param name="value">The <see cref="sbyte"/> to write.</param>
+        /// <param name="cancellationToken">The token to cancel the operation. The default value is None.</param>
+        public static async Task WriteSByteAsync(this Stream stream, sbyte value,
+            CancellationToken cancellationToken = default)
+        {
+            await stream.WriteAsync(new[] {(byte)value}, 0, 1, cancellationToken).ConfigureAwait(false);
+        }
```

Comment:
Can this delegate to existing WriteByteAsync?
```suggestion
        public static async Task WriteSByteAsync(this Stream stream, sbyte value,
            CancellationToken cancellationToken = default)
        {
            await stream.WriteByteAsync((byte)value, cancellationToken).ConfigureAwait(false);
        }
```